File: C++/ABC131/ABC131A.cpp
#include <iostream>
#include <string>
#include <vector>
#include <algorithm>

using namespace std;


int main(){

    string str;
    cin>>str;

    vector<int> nums(str.size());

    transform(str.begin(),str.end(),nums.begin(),[](char ch) {return ch- '0';});

    bool flag=true;
    int before_num;

    for(int i=0;i<nums.size();i++){
        if(flag){
            if(i==0) before_num=nums[i];
            else{
                if(nums[i]==before_num){
                    flag=false;
                    cout<<"Bad"<<endl;
                }
                before_num=nums[i];
            }
        }
    }

    if(flag) cout<<"Good"<<endl;

    //for(int i=0;i<nums.size();i++) cout<<nums[i];

    return 0;
}

File: C++/ABC131/ABC131A.py

num=str(input())

lst=[]

while num>0:
    lst.append(num%10)
    num//=10
lst.reverse()

l=0
flag=True
for index,i_num in enumerate(lst):
    print(index,i_num)
    if flag:
        if index==0:
            l=i_num
        else:
            if i_num==l:
                print("Bad")
                flag=False
                break
            l=i_num

if flag:
    print("Good")            
